examples/PalmerPenguins.jl

```diff
@@ -0,0 +1,65 @@
+using PalmerPenguins, Plots
+const TABLE = PalmerPenguins.load()
+
+using DataFrames, Random
+df = dropmissing(DataFrame(TABLE))
+df[randperm(nrow(df)), :]
+first(df, 5)
+
+# Binary classification using Bernoulli likelihood - sex
+
+unique(df[:sex])
+
+
+using KernelFunctions, AbstractGPs, GPLikelihoods
+x = RowVecs(Array(df[:, [:bill_length_mm, :bill_depth_mm, :flipper_length_mm, :body_mass_g]]))
+y = [sex=="female" ? true : false for sex in df[:sex]]
+
+scatter(
+    df[:bill_length_mm],
+    df[:bill_depth_mm],
+    df[:flipper_length_mm],
+    group = df[:species],
+    m = (0.5, [:+ :h :star7], 5),
+)
+
+x_train, y_train = x[1:266], y[1:266]
+x_test, y_test = x[267:end], y[267:end]
+
+
+k = Matern52Kernel()
+f = LatentGP(GP(k), BernoulliLikelihood(), 0.001)
+fx = f(x_train)
+logpdf(fx, rand(fx))
```

Comment:
I'm not sure what you want to compute here. I assume you would rather want to base your evaluation on e.g. `x_test` and `y_test`? However, a general issue is that you can evaluate `logpdf` only for a named tuple of `y` and `f` (the outputs in the GP) but only `y` is available and `f` is a latent variable. So either you would have to sample one specific `f` from the GP or marginalize out `f` (which is usually not possible in closed form).